aana/deployments/whisper_deployment.py

```diff
@@ -182,27 +181,12 @@ async def transcribe(
         Raises:
             InferenceException: If the inference fails.
         """
-        if not params:
-            params = WhisperParams()
-
-        audio_array = audio.get_numpy()
-        if not audio_array.any():
-            # For silent audios/no audio tracks, return empty output with language as silence
-            return WhisperOutput(
-                segments=[],
-                transcription_info=AsrTranscriptionInfo(
-                    language="silence", language_confidence=1.0
-                ),
-                transcription=AsrTranscription(text=""),
-            )
-
-        try:
-            segments, info = self.model.transcribe(audio_array, **params.model_dump())
-        except Exception as e:
-            raise InferenceException(self.model_name) from e
+        asr_segments = []
+        asr_transcription_info = None
```

Comment:
L188: can't we do the following?
```
if not asr_transcription_info:
       asr_transcription_info = output["transcription_info"]
```
There will only be one assignment in this case.